db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/SPSDescriptor.java

```diff
@@ -911,12 +911,18 @@ private void updateSYSSTATEMENTS(LanguageConnectionContext lcc, int mode, Transa
         if (tc == null) { //bug 4821 - tc will passed null if we want to use the user transaction
             tc = lcc.getTransactionExecute();
         }
+        try {
+            dd.updateSPS(this,
+                    tc,
+                    recompile,
+                    updateSYSCOLUMNS,
+                    firstCompilation);
+
+        } catch (StandardException se) {
+            // This can fail since multiple threads/clusters can recompile sps descriptors
+            // especially after collecting stats on sys tables.
```

Comment:
Could you check it is a WW conflict and continue raising the exception otherwise?